library/spdm_responder_lib/libspdm_rsp_algorithms.c

```diff
@@ -262,6 +265,117 @@ libspdm_return_t libspdm_get_response_algorithms(libspdm_context_t *spdm_context
         SPDM_MEL_SPECIFICATION_DMTF,
     };
 
+    uint32_t pqc_asym_priority_table[] = {
```

Comment:
Should probably run this by the SPDM Working Group.

Reply:
I am fine to ask. But I am not sure what answer we will get...

Overall this is policy driven. Maybe we can expose an priority table to allow integrator to setup.